docs/aspects.md

```diff
@@ -0,0 +1,78 @@
+# All About Aspects
+
+Aspects codify features of the world, the environment, the situation, and your character. They are simple, plain truths that describe both the physical and the abstract. 
+
+There are two primary ways to use Aspects: Invocation and Compels.
+
+## Clarifications
+
+### "Aspects Are True"
+
+Because Aspects Are True, any object or condition described by an Aspect carries narrative weight and sometimes mechanical impact. If you have taken a Consequence for a ***Broken Hand***, it's not just for flavor; you probably cannot handwrite a letter. 
+
+Likewise, any significant setpieces or rules may become codified as an Aspect with the GM's discretion. Though, not every single little thing needs to become an Aspect. 
+
+Of course, this also means that once an Aspect *stops* being true, the Aspect (and any of its Free Invokes) can disappear. If a building was ***On Fire*** but no longer is, then you can no longer invoke that Aspect.
+
+### Boosts 
+
+Boosts are temporary type of Aspect usually granted as a bonus or a consolation under the [Four Actions](Four-Actions)' outcomes. Boosts have 1 Free Invoke, but disappear as soon as you use them.
+
+Thus, Boosts are best named for temporary things, such as a ***Burst of Energy*** or ***Dropped Guard***.
```

Comment:
Move this after Using Aspects -- your last paragraph just set up the two ways to use Aspects